guide/sections/part2/wis2-architecture.adoc

```diff
@@ -112,7 +113,14 @@ Refer to the _Manual on WIS_, Volume II for details.
 * A Global Discovery Catalogue amends discovery metadata records to add details of where one can subscribe to updates about the dataset at a Global Broker.
 * A Global Discovery Catalogue makes its content available for indexing by search engines.
 
-==== 2.4.5 Global Monitor
+==== 2.4.5 Global Replay
+* WIS2 may include a Global Replay.
+* A Global Replay enables Global Services and data consumers to search, query and subscribe to notification messages and monitoring events published by the Global Broker.
+* A Global Replay subscribes to notification messages and monitoring events via a Global Broker about the availability of new notification messages. It downloads a copy of the notification message and updates its local index.
+* A Global Replay shall retain copies of notification messages and monitoring events for a duration compatible with the real-time or near real-time schedule of the data and not less than 24-hours.
+* A Global Replay will delete notification messages and monitoring events from its local index once the retention period has expired.
```

Comment:
A Global Replay will discard notification messages and monitoring events once the retention period has expired.